src/Pages/ItemDetail/Components/InfoAndCartPut.js

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable prettier/prettier */
+import React, { Component } from "react";
+import "./InfoAndCartPut.scss";
+
+const itemData = {
+  id: 1,
+  name: "무농약 토마토",
+  shortDescription: "친환경으로 재배한 신선한 무농약 토마토",
+  price: 7650,
+  sellUnit: "1개",
+  weight: "1kg",
+  deliveryType: "샛별배송/택배배송",
+  originCountry: "국산",
+  packageType: "냉장/종이포장",
+  allergyInfo: [
+    "소고기 함유",
+    "본 제품은 돼지고기를 사용한 제품과 같은 제조시설에서 제조했습니다.",
+  ],
+  shelfLife: "수령일 포함 4일 이상 남은 상품을 보내드립니다.",
+};
+const allergyInfo = [];
+for (let i = 0; i < itemData.allergyInfo.length; i++) {
+  allergyInfo.push(itemData.allergyInfo[i], <br />);
+}
+
+const itemDetailInfo = [
+  { title: "판매단위", content: itemData.sellUnit },
+  { title: "중량/용량", content: itemData.weight },
+  { title: "배송구분", content: itemData.deliveryType },
+  { title: "원산지", content: itemData.originCountry },
+  { title: "포장타입", content: itemData.packageType },
+  {
+    title: "유통기한",
+    content: itemData.shelfLife,
+  },
+  {
+    title: "알레르기 정보",
+    content: allergyInfo,
+  },
+];
+class InfoAndCartPut extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      quantity: 1,
+    };
+  }
+
+  handleQuantity = e => {
+    const QUANTITY_MIN = 1;
+    const currentQuantity = this.state.quantity;
+    const type = e.target.className;
+    if (currentQuantity === QUANTITY_MIN && type === "subtract") return;
+    this.setState({
+      quantity: type === "add" ? currentQuantity + 1 : currentQuantity - 1,
+    });
+  };
+
+  render() {
+    const { quantity } = this.state;
+    const userMileageClass = 0.005;
+    const mileage = Math.ceil(itemData.price * userMileageClass * quantity);
+    return (
+      <div className="InfoAndCartPut">
+        <div className="item-image">
+          <img src="images/tomato.jpg" alt="tomato"></img>
+        </div>
+        <div className="item-detail-right">
+          <div className="item-info">
+            <div className="item-name">
```

Comment:
우리는 이제 sass의 nesting 기능을 이용합니다.그래서 className을 작성하실 때, 굳이 부모 태그의 className을 작성하면서 태그 하나하나 이름을 분리해 주지 않아도,nesting 기능을 통해 css 속성이 분리될 수 있습니다.그래서 위와 같은 태그의 경우도 아래와 같이 간단하게 className을 작성해주실 수 있습니다.

다른 className들도 nesting기능을 이용하니, 간단하게 변경해보세요!
```suggestion
        <div className="detailRight">
          <div className="info">
            <div className="name">
```

Reply:
이름 짓기가 항상 힘드네요 ㅠㅠ
이제 scss에 적응해서 위와 같이 반영하겠습니다!